src/main/java/edu/harvard/iq/dataverse/ingest/plugin/spi/DigraphNode.java

```diff
@@ -66,16 +66,16 @@
  * the current node) may be queried.
  *
  */
-class DigraphNode implements Cloneable, Serializable {
+class DigraphNode<E> implements Cloneable, Serializable {
```

Comment:
This set of changes is the one part of this that I'm not completely sure is valid. Can someone look over this to make sure I got the E's right?